### PR TITLE
Diona no longer consumes air to restore nutrition.

### PR DESCRIPTION
**Pull request by Sindorman**
Apparently Diona was consuming air in order to restore nutrition. And light would not restore it at all.

- Diona no longer consumes air to restore nutrition, instead it uses light. Was requested by Neinbox.

### CODE REVIEW

**Comment by BurgerLUA** [comment]
I thought it was well known in lore that diona would consume air to harvest passing nutrients at peak efficiency.

**Comment by Sindorman** [comment]
!review

File: code/modules/mob/living/carbon/carbon_defines.dm [hunk 1]
```diff
@@ -33,4 +33,5 @@
 
 	var/coughedtime = null // should only be useful for carbons as the only thing using it has a carbon arg.
 
-	var/willfully_sleeping = 0
\ No newline at end of file
+	var/willfully_sleeping = 0
+	var/consume_nutrition_from_air = FALSE // used by Diona
```
Comment (skull132) [inline code/modules/mob/living/carbon/carbon_defines.dm]: Isn't there a "diona datum" thing where this variable could be off-loaded to?

Comment (skull132) [reply code/modules/mob/living/carbon/carbon_defines.dm]: Yes, there is. "diona/datum".

Comment (Sindorman) [reply code/modules/mob/living/carbon/carbon_defines.dm]: It is also used by Diona nymph which is of carbon type and doesn't have diona datum iirc

Comment (skull132) [reply code/modules/mob/living/carbon/carbon_defines.dm]: _Aaaaaaaaaaaaaaaaaaaaaa._

Comment (skull132) [reply code/modules/mob/living/carbon/carbon_defines.dm]: I repeat my statement. _Aaaaaaaaaaaaaaaaaaaaaaaa._

**Pull request merged**